### DISCUSSION TITLE
Orca slicer support for Bcn3D

### DISCUSSION THREAD

**Discussion by JirkaFish** [OrcaSlicer/OrcaSlicer#4991] [Dev]
Hello,
At work we have a 3d printer from Bcn3D type Epsilon W50.
I would welcome support for this printer in Orca slicer.
I tried to modify the start and end Gcode and adjust the settings in Orca slicer to be able to slice files for Epsilon W50 but without success.
Could you give me some advice?
I am adding a .gcode file that is functional.
Thank you

[EW50_Pravy naraznik_Dil_B.gcode.txt](https://github.com/SoftFever/OrcaSlicer/files/14967217/EW50_Pravy.naraznik_Dil_B.gcode.txt)

**Comment by ghost** [comment]
@JirkaFish I am trying to do the same with a W50. If you have any insight or are willing to share printer config I would really appreciate it. Thanks!

**Comment by GarretWayne** [comment]
Hello there!

Any news on this? Would love to use Orca with our Epsilon W50, and we could also help with the configs. Recently, we had an Orca update to support IDEX printers, so I'll set up a profile and go from there first.

Also, I found this (haven't tried it yet) : https://www.printables.com/model/1113910-epsilon-w50-orca-slicer-pla-print-profile_orca-cub/files

**Comment by Okayler** [comment]
I have tried today some things to integrate the d25 into the orcaslicer:
https://github.com/Okayler/orcaslicer-BCN3DSigmaD25/tree/main

A first benchy is printed. When you want you can take it and modify and contribute :)
